libcontainer/system/proc.go

```diff
@@ -96,7 +96,7 @@ func parseStat(data string) (stat Stat_t, err error) {
 	// one (PID) and two (Name) in the paren-split.
 	parts = strings.Split(data[i+2:], " ")
 	var state int
-	fmt.Sscanf(parts[3-3], "%c", &state)
+	fmt.Sscanf(parts[3-3], "%c", &state) //nolint:staticcheck // "3-3" is more readable in this context.
```

Comment:
I have a fix for this https://github.com/opencontainers/runc/pull/2696 (which started as a fix for this linter issue but ended up as a rewrite of `parseStat` to be 7x faster and more error-prone).